code_samples/node_modules/ssl_sample.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var Config = require('hazelcast-client').Config;
```

Comment:
Since you are going over all the code samples, try to be consistent. In your SQL sample, you imported `Config.ClientConfig` but for this `Config` is imported. For SQL sample you used  `var Client = require('hazelcast-client').Client;` but for this you used `var HazelcastClient = require('hazelcast-client').Client;`. Pick one of them and it stick to it.